grpc/estimator_service.h

```diff
@@ -40,47 +39,47 @@ class EstimatorService final : public estimator::Estimator::Service {
 
   grpc::Status Init(grpc::ServerContext* context,
                     const estimator::InitRequest* request,
-                    estimator::InitResponse* response) override;
+                    estimator::InitResponse* response);
```

Comment:
what happened to all the `override` statements? There should either be `override` or `final`.